src/components/youtube/embed.tsx

```diff
@@ -1,18 +1,20 @@
 // Youtube LoFi Girl Embed
+import moment from 'moment-timezone';
 
 export default function YoutubeEmbed() {
-  // Returns a youtube embed if it is Thursday 7:00-9:00pm, otherwise empty div.
-  const date = new Date();
-  const day = date.getDay();
-  const hour = date.getHours();
-  const minute = date.getMinutes();
-  if (day !== 4) {
+  // Tuesday or Thursday between 6pm and 8pm UK time
+  const ukTime = moment().tz('Europe/London');
+  const day = ukTime.day();
+  const hour = ukTime.hour();
+  const minute = ukTime.minute();
+
+  if (day !== 2 && day !== 4) {
```

Comment:
can you elaborate?

Reply:
can you elaborate?

Reply:
Elaboration: don't do code review at 3:40am. It's all good, and works I think. Mb lol

Reply:
I will say tho that these ifs take a bit of brain power to figure out... Is it not worth inverting them into 1 if that returns the correct output, and then an else for <div />? 

It means you'd be filtering to when you want code night, rather than when you don't want code night. May be nicer to read? Either way, this works, so ignore 👍👍👍

Reply:
The practice of using guard clauses - returning early for the "unhappy paths" tends to reduce nesting. I moved this away into a function instead to require less cognitive load when first looking at the yt embed code.